voxblox/CMakeLists.txt

```diff
@@ -54,9 +54,22 @@ cs_add_library(${PROJECT_NAME}
 )
 target_link_libraries(${PROJECT_NAME} ${PROTOBUF_LIBRARIES})
 
-#########
-# TESTS #
-#########
+##############
+# BENCHMARKS #
+##############
+find_package(benchmark_helpers REQUIRED)
+add_benchmark(bm_cast_ray test/benchmark_cast_ray.cc)
```

Comment:
with this cmake macros it is now possible to run benchmark just as gtests with:
make run_benchmarks: runs all
make run_benchmarks_NAME: runs only this benchmark
make benchmarks: builds them all
...